package.json

```diff
@@ -16,12 +16,12 @@
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "percy exec -- ember test"
   },
   "devDependencies": {
     "@ember/optional-features": "^1.3.0",
     "@ember/render-modifiers": "^1.0.2",
-    "@sinonjs/fake-timers": "^6.0.1",
```

Comment:
I think this package was used as a workaround for Percy v1.x.

Reply:
👍  yeah it can come out